portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
```

Comment:
Use your own judgement.
I wonder if it worth a new file.
It looks to me that it'll only be used by Servlet.
You could define multiple classes in one file as long as only one class is public.